docs/content/any/examples/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Examples
+weight: 3
+isAny: true
+---
```

Comment:
should we add an alias for the deleted `docs/content/any/guides/more/_index.md` index here?

Reply:
No it's very unlikely there are any links to that page.